projects/storefrontlib/src/cms-components/product/product-tabs/product-details-tab/product-details-tab.component.ts

```diff
@@ -10,10 +10,14 @@ import { CurrentProductService } from '../../current-product.service';
 })
 export class ProductDetailsTabComponent implements OnInit {
   product$: Observable<Product>;
+  showMore: boolean;
```

Comment:
better give it a default value: `showMore=false`

Reply:
done